docs/core/community.md

```diff
@@ -8,31 +8,51 @@ We welcome issues, contributions and discussion from all users, regardless of ba
 
 Please consider [Chris White](https://github.com/cicdw) the main point of contact for the Prefect repo.
 
-## Discussion
+# Slack
 
-- We have a public [Slack](https://www.prefect.io/slack) for chatting about Prefect, asking questions, and sharing tips.
-- For feature requests or bug reports, please open a [new GitHub issue](https://github.com/PrefectHQ/prefect/issues/new).
-- Important architectural and design decisions are memorialized in [Prefect Improvement Notices (PINs)](/core/PINs/PIN-01-Introduce-PINs.html).
+We have a public [Slack](https://prefect-slackin.herokuapp.com/) for chatting about Prefect, asking questions, and sharing tips.
```

Comment:
Is this the heroku app that we just replaced? cc @znicholasbrown

Reply:
Yup it is - use this instead:
```suggestion
We have a public [Slack](https://prefect.io/slack) for chatting about Prefect, asking questions, and sharing tips.
```